docs/src/modules/java/pages/shopping-cart-quickstart.adoc

```diff
@@ -217,30 +217,6 @@ Here, you can view not only the http://localhost:3000/services/shopping-cart-qui
 akka auth login
 ----
 
-. Create a project:
-+
-[source, command line]
-----
-akka projects new my-first-app --region aws-us-east-2
-----
-
-. Observe that your project was created successfully:
-+
-[source]
-----
-NAME          DESCRIPTION   ID                                     OWNER                   REGION
-my-first-app                703391f5-cee4-4d50-8a7f-07c5d1183aaa   your-organization-name  aws-us-east-2
-
-To make this the currently active project, run: 'akka config set project my-first-app'
-----
-
-. Make the project you just created the currently active project:
-+
-[source, command line]
-----
-akka config set project my-first-app
```

Comment:
IIUC, the CLI automatically makes `default-project` the target for its commands, right?

Reply:
Yes